core/src/main/java/co/aikar/commands/BaseCommand.java

```diff
@@ -479,26 +695,62 @@ private void executeCommand(CommandOperationContext commandOperationContext,
         }
     }
 
+    /**
+     * Whether the command can be executed by the issuer.
+     *
+     * @param issuer
+     *         The issuer to take into account.
+     * @param cmd
+     *         The command to check whether the issuer can execute.
+     *
+     * @return Whether the issuer can execute the command.
+     */
     public boolean canExecute(CommandIssuer issuer, RegisteredCommand<?> cmd) {
-        return true;
+        return cmd.hasPermission(issuer);
```

Comment:
not needed. permission checks were already done. this was old api that will prob go away and meant to be used for misc stuff.